osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,29 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "chat":
+                case "msg":
+                case "query":
+                    if (string.IsNullOrWhiteSpace(content))
+                    {
+                        target.AddNewMessages(new ErrorMessage($"Usage: /{command} [user]"));
+                        break;
+                    }
+
+                    // Check if the user has joined requested channel already.
+                    var alreadyJoinedChannel = JoinedChannels.FirstOrDefault(
+                        c => c.Type == ChannelType.PM && c.Users.Count == 1 && c.Name.Equals(content, StringComparison.OrdinalIgnoreCase));
```

Comment:
Could there be a PM with more than one user? otherwise this is unnecessary.

```suggestion
                    var privateChannel = JoinedChannels.SingleOrDefault(c => c.Type == ChannelType.PM && c.Name.Equals(content, StringComparison.OrdinalIgnoreCase));
```

Reply:
I copied it from `OpenPrivateChannel` which uses
```cs
CurrentChannel.Value = JoinedChannels.FirstOrDefault(c => c.Type == ChannelType.PM && c.Users.Count == 1 && c.Users.Any(u => u.Id == user.Id))
                       ?? JoinChannel(new Channel(user));
```
might leave the user count check in just in case? maybe something was planned for the future?

Reply:
Sure I guess.